modules/login/php/requestaccount.class.inc

```diff
@@ -269,7 +269,67 @@ class RequestAccount extends \NDB_Form
             }
         }
 
+        // Verify reCAPTCHA
+        $err = [];
+        if (isset($values['g-recaptcha-response']) && isset($reCAPTCHAPrivate)) {
+            $recaptcha = new \ReCaptcha\ReCaptcha($reCAPTCHAPrivate);
+            $resp      = $recaptcha->verify(
+                $values['g-recaptcha-response'],
+                $_SERVER['REMOTE_ADDR']
+            );
+            if (!$resp->isSuccess()) {
+                $err['captcha'] = 'Please complete the reCaptcha!';
+            }
+        }
         // Show success message even if email already exists for security reasons
         $this->tpl_data['success'] = true;
+
+        // If there are error messages, return early without sending emails.
+        if (!empty($err)) {
+            return;
+        }
+
+        // Send an email to all users meeting the following criteria:
+        //     - Associated with the Site
+        //     - Associated with the Project
+        //     - Have the User Management permission
+        $query  = <<<SQL
+        SELECT DISTINCT u.Email,u.userID 
+        FROM users u 
+        JOIN user_psc_rel upr 
+        ON u.ID = upr.UserID 
+        JOIN user_project_rel uprojr 
+        ON u.ID = uprojr.UserID 
+        WHERE upr.CenterID=:center
+        AND uprojr.ProjectID=:project
+SQL;
+        $result = $DB->pselect(
+            $query,
+            ['center' => $site, 'project' => $project]
+        );
+        foreach ($result as $row) {
+            $hasPermission = \User::factory($row['userID'])
+                ->hasAnyPermission(
+                    ['user_accounts', 'user_accounts_multisite']
+                );
+            if ($hasPermission === false) {
+                continue;
+            }
+
+            // Send email to users with User Management permissions.
+            $config    = \NDB_Factory::singleton()->config();
+            $studyName = $config->getSetting('title');
+
+            \Email::send(
```

Comment:
I wont block this PR over this but I think we should use the notifier class in the future for this

Reply:
How much work is involved in changing it to use the Notifier class? That seems like a reasonable change..

Reply:
not that much but it doesn't bring many benefits.. other I think than being able to subscribe or unsubscribe from email notifications

Reply:
https://github.com/aces/Loris/wiki/Notification-system

Reply:
That could be future work. You could make a ticket to refactor this to use the notification system

Reply:
@ridz1208 is there an issue already created for that? If not, would you mind creating it since you know what you are talking about and I don't? Thank you! :)